frontend/src/components/StakeSelectorItem.vue

```diff
@@ -24,10 +24,11 @@
         </tr>
         <tr v-if="estimatedYield" :title="$t('stake.StakeSelectorItem.yieldTooltip')">
           <th class="bold">
-            {{$t('stake.StakeSelectorItem.apy')}}
+            {{ isNftStaking ? $t('stake.StakeSelectorItem.yield') : $t('stake.StakeSelectorItem.apy') }}
           </th>
           <td class="align-right">
-            {{ estimatedYield.multipliedBy(100).toFixed(2) }}%
+            {{ isNftStaking ? estimatedYield.toFixed(2) : estimatedYield.multipliedBy(100).toFixed(2) }}
+            {{ isNftStaking ? '/NFT' : '%' }}
```

Comment:
I would opt for parent div (in  this case td) to be rendered based on isNftStaking condition, to not check the same thing twice. But it's minor, do it, or not, we're good :)